commit 6b0a8a22773c82160a49c0865f421c306eb31836
Author: Matthew Booth <mbooth@redhat.com>
Date:   Wed Oct 17 10:43:18 2012 +0100

    libvirt: Create disks with cache=none
    
    Fixes RHBZ#838057
    (cherry picked from commit d907c8a260daa0be5347bccea404d8977edd830c)

diff --git a/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm b/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm
index cf9c78b..20a4ced 100644
--- a/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm
+++ b/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm
@@ -407,6 +407,7 @@ DOM
         my $driver = _append_elem($diskE, 'driver');
         $driver->setAttribute('name', 'qemu');
         $driver->setAttribute('type', $disk->{dst}->get_format());
+        $driver->setAttribute('cache', 'none');
 
         my $source = _append_elem($diskE, 'source');
         $source->setAttribute($is_block ? 'dev' : 'file',
